javascript-typing-tutor/script.js

```diff
@@ -0,0 +1,37 @@
+var value = 0;
+var attempts = 0;
+var $text = document.querySelectorAll('span');
+var $score = document.querySelector('p');
+
+document.addEventListener('keydown', function (e) {
+  if (e.key === $text[value].innerText) {
+    $text[value].className = 'green-text';
+    attempts++;
+    value++;
+    $score.innerHTML = 'Score: ' + value + '/27';
```

Comment:
Using the `innerHTML` property is a big no-no. It leaves you open to script injections, which is a bad security risk. Instead, you should put an element like a `span` around where the score would be in this element, then just update the `textContent` of that one element.